specification/app/resource-manager/Microsoft.App/preview/2025-10-02-preview/ContainerAppFunctions.json

```diff
@@ -0,0 +1,414 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2025-10-02-preview",
+    "title": "ContainerApps API Client",
+    "description": "Container App Functions API for managing functions within Container Apps and their revisions."
```

Comment:
These are read only APIs right? I don't see any write operations to call it "managing" 😅